pkg/assume/assume.go

```diff
@@ -121,6 +121,15 @@ func AssumeCommand(c *cli.Context) error {
 	}
 
 	configOpts := cfaws.ConfigOpts{Duration: time.Hour}
+
+	//attempt to get session duration from profile
+	defaultDuration, err := profile.DefaultDuration(c.Context)
```

Comment:
I propose that yuou can remove this fuctiuon call and jist use 
```suggestion
	if profile.AWSConfig.RuleDurationSeconds != nil {
	configOpts.Duration = *profile.AWSConfig.RuleDurationSeconds
	}
```